config/settings.yml

```diff
@@ -0,0 +1,46 @@
+---
+:ems:
+  :ems_amazon:
+    :disabled_regions: []
+    # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
+    # uncomment the following to disable gov-west-1 region
```

Comment:
I would remove this statement.  We don't want *users* editing this file at all.  Instead they would go either go through Advanced Settings (which stores the changes in the database) or we would have them deploy a settings.local.yml or something.